server/superdesk/io/rss.py

```diff
@@ -104,23 +127,32 @@ def _fetch_data(self, config, provider):
                 raise IngestApiError.apiGeneralError(
                     Exception(response.reason), provider)
 
-    def _create_item(self, data):
+    def _create_item(self, data, field_aliases=None):
         """Create a new content item from RSS feed data.
 
         :param dict data: parsed data of a single feed entry
+        :param field_aliases: (optional) field name aliases. Used for content
+             fields that are named differently in retrieved data.
+        :type field_aliases: dict or None
+
         :return: created content item
         :rtype: dict
         """
-        item = dict()
-        item['guid'] = item['uri'] = data.get('guid')
-        item['type'] = 'text'
-        if data.get('updated_parsed'):
-            item['versioncreated'] = utcfromtimestamp(timegm(data.get('updated_parsed')))
-        if data.get('published_parsed') or data.get('updated_parsed'):
-            item['firstcreated'] = utcfromtimestamp(timegm(data.get('published_parsed', data.get('updated_parsed'))))
```

Comment:
@amagdas This behavior has slightly changed, the `firstcreated` field is now only generated from the `published_parsed` field in data, but not from `updated_parsed` - is that OK now that we can define field name aliases?